src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetricsOptions.cs

```diff
@@ -50,6 +50,11 @@ public class RuntimeMetricsOptions
         /// </summary>
         public bool? AssembliesEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether exception metrics should be collected.
+        /// </summary>
+        public bool? ExceptionsEnabled { get; set; }
```

Comment:
```suggestion
        public bool? ExceptionCountEnabled { get; set; }
```

Or we have more metrics related to Exceptions?

Reply:
The naming style is the same as other flags: `GcEnabled`, `JitEnabled`, `ThreadingEnabled` and `AssembliesEnabled`. If it needs change because it's confusing, at least assembly count also needs changing.

Do `AssemblyCountEnabled` and `ExceptionCountEnabled` look good?

Reply:
Random thought, but I wonder if we should maybe use enums w/ `[Flags]` to give more granular control of these? Off the dome thinking something like...

```csharp
[Flags]
public enum JitMetricTypes : int
{
   None = 0x00,
   All = IlBytesJitted | MethodsJittedCount | TimeInJit ,

   TotalBytes = 0b1,
   MethodCount = 0b10,
   ProcessingTime = 0b100,
}

[Flags]
public enum ExceptionMetricTypes : int
{
   None = 0x00,
   All = Count,

   Count = 0b1
}
```

Reply:
This looks nice. We should have a separate dedicated discussion about the naming/enable/disable options for this package before stable release.

Reply:
I've thought about the bitwise combination as well. I felt it might be counterintuitive to have `.AddRuntimeMetrics()` as enabling all options as default, and after adding an option, say `.AddRuntimeMetrics(options => { options.GcEnabled = true; })`, all options other than `GcEnabled` will be disabled. However, each of the two still makes sense when seen alone, so I didn't just went ahead and change it.

One thing to note is `int` type only has 32 bits (or 64 for `long`), having too much granularity will hit the cap soon. Separating it into sections will make better sense if we do it the bitwise combination way.